nucliadb/nucliadb/search/search/query.py

```diff
@@ -372,7 +432,16 @@ async def parse_relation_search(
         autofilters = []
         relations_search = SearchOptions.RELATIONS in self.features
         if relations_search or self.autofilter:
-            detected_entities = await self._get_detected_entities()
+            if self.query_endpoint_enabled:
+                detected_entities = await self._get_detected_entities()
+            else:
+                query_info_result = await self._get_query_information()
```

Comment:
aren't you awaiting it twice? one here and one in line 401?

Reply:
No its a common strategy to wait multiple times the same task: https://stackoverflow.com/questions/71970942/can-i-await-the-same-task-multiple-times-in-python